src/build/mod.rs

```diff
@@ -307,6 +306,34 @@ fn reproducible_build_directives() -> Vec<Directive> {
     ]
 }
 
+pub fn build_user_service(entrypoint: String, wait_for_env: &str, last_user: String) -> Directive {
+    let su_cmd = format!("su {last_user}");
+    let exec_cmd = format!("exec {}", entrypoint);
+
+    let cmds = vec![
+        su_cmd.as_str(),
+        "sleep 5",
+        "echo \\\"Checking status of data-plane\\\"",
```

Comment:
interested to know why these need to be triple escaped? (just copied from og implementation)

Reply:
it's just escaping the two characters (`/` -> `\\` and `"` -> `\"`). The strings here are then printed to a file in the Cage, which results in `echo "..."` being written.

Would be cleaner to use raw strings tho:
```suggestion
        r#"echo \"Checking status of data-plane\""#,
```